src/_includes/soft_mod_hubspot_in_page_banner_cta.liquid

```diff
@@ -0,0 +1,19 @@
+{%- if site.lang == 'es' -%}
+  <span class="hs-cta-wrapper hubspot-in-page-banner" id="hs-cta-wrapper-a4f30da8-3062-4c47-a865-a04a70bfa951">
```

Comment:
I guess that `hs-cta-wrapper` comes from the embed code from HubSpot.

Should we remove it because we're styling the CTAs ourselves?

Reply:
This has been amended in the following commit [ced81b9](https://github.com/codurance/site/commit/ced81b9efd4fa5d4a08c5ffbf51ed50798e173b3)